packages/forge/blocks/zendeskChat/actions/openMessenger.ts

```diff
@@ -0,0 +1,65 @@
+import { createAction, option } from '@typebot.io/forge'
+import { auth } from '../auth'
+
+export const openMessenger = createAction({
+  auth,
+  name: 'Open Messenger',
```

Comment:
Is Messenger the right word. Is it what Zendesk using?

Reply:
Zendesk is a bit confusing here. Messaging is a Channel for them and their Web Widget is one of the UIs available for Messaging, along with others like FB Messenger, WhatsApp, etc.

<img width="1451" alt="Screenshot 2024-09-10 at 10 29 53 PM" src="https://github.com/user-attachments/assets/7d36c69d-9bcb-4131-869c-b7cf8eddfc80">

<img width="1448" alt="Screenshot 2024-09-10 at 10 27 24 PM" src="https://github.com/user-attachments/assets/e6d006da-c533-4394-bf45-7d97bd84968f">


So, this could be changed to openWebWidget. We could also add additional actions for the other messaging channels in the future.

Reply:
renamed action to openWebWidget